sisl/io/tests/test_object.py

```diff
@@ -1,14 +1,12 @@
 from __future__ import print_function, division
 
+import tempfile
```

Comment:
Instead of using `tempfile` you should add an argument `sisl_tmp` to the test which requires a temporary file.
That would ensure you have a temporary file based on the same test-directory. See below